assets/src/blocks/Spreadsheet/Spreadsheet.js

```diff
@@ -16,22 +16,38 @@ export class Spreadsheet extends Component {
   renderEdit() {
     const { __ } = wp.i18n;
 
-    const { color } = this.props;
+    const { attributes, setAttributes } = this.props;
+
+    const toAttribute = attributeName => value => {
+      setAttributes( { [ attributeName ]: value } );
+    };
 
     const colors = [
       { name: 'blue', color: '#C9E7FA' },
       { name: 'green', color: '#D0FAC9' },
       { name: 'grey', color: '#DCDCDC' },
     ];
 
+    const toCssVariable = ( variableName ) => ( value ) => {
```

Comment:
Probably this function can be extracted so it can be reused in other blocks. But maybe it makes more sense to do once we implement the same in another block?